.github/workflows/test_build_docker_image.yaml

```diff
@@ -0,0 +1,62 @@
+name: "Test build Docker image"
+
+env:
+  DEFAULT_PYTHON_VERSION: "3.10"
+  FORCE_COLOR: "1" # Make tools pretty.
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+
+# ensuring only one instance is running at a given time
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
This now has to include the workflow. Otherwise, it would not be possible to run multiple workflows concurrently.